developer.md

```diff
@@ -8,37 +8,61 @@ git clone --recursive https://github.com/RediSearch/RediSearch.git
 
 ## Installing prerequisites
 
-## Build dependencies
+### Build dependencies
 To build and test RediSearch you need to install several packages, depending on the underlying OS. The following OSes are supported and tested in our CI:
 
-Ubuntu 18.04
-Ubuntu 20.04
-Ubuntu 22.04
-Ubuntu 24.04
-Debian linux 11
-Debian linux 12
-Rocky linux 8
-Rocky linux 9
-Amazon linux 2
-Amazon linux 2023
-Mariner 2.0
-MacOS
-Alpine linux 3
+* Ubuntu 18.04
+* Ubuntu 20.04
+* Ubuntu 22.04
+* Ubuntu 24.04
+* Debian linux 11
+* Debian linux 12
+* Rocky linux 8
+* Rocky linux 9
+* Amazon linux 2
+* Amazon linux 2023
+* Mariner 2.0
+* MacOS
+* Alpine linux 3
 
 For installing the prerequisites you can take the following approaches:
-1. Install the dependencies manually - you can find the dependencies in each OS script under the [`.install`](.install) directory.
+1. Install the dependencies manually - check our install script at ".install/install_script.sh" for the list of dependencies. The script is designed to be run in a Docker environment, so you can use it as a reference for installing the dependencies manually.  In general, the common installations are:
+   - `rust` (latest stable version)
+   - `cmake >= 3.25.1`
+   - `boost == 1.84.0`
```

Comment:
not sure we actually tied to these specific versions
on all OSes (not including make) i just installed the default cmake provided by the os package manager
Regarding boost, it will be installed during make build process, maybe worth mentioning.